oasislmf/model_execution/bin.py

```diff
@@ -96,11 +97,11 @@ def prepare_run_directory(
     Darwin or Linux, otherwise the source folder tree is recursively
     copied into the ``static`` subfolder.
 
-    :param run_directory: the model run directory
-    :type run_directory: str
+    :param run_dir: the model run directory
```

Comment:
Interesting that the Sphinx style docstrings seem to be the norm is this repository, in contrast to the Google style docstrings used in the platform repository.

Reply:
Yes, hence why I think that one you spotted elsewhere got through - it's quite easy to get mixed up when you're swapping back and forth between two or three projects.

Reply:
MDK uses Sphinx for its docs, hence the reST syntax for docstrings. This will continue to be the case. The Google style docstring format used in Oasis Platform may be more appropriate there because of the requirements for documenting the API. There's no reason why the two have to be the same.

Reply:
Consistency between projects would be nice but if there are technical reasons for doing things differently in different projects we'll respect that.